include/macros.h

```diff
@@ -31,8 +31,11 @@
 
 #define GET_ACTIVE_CAM(globalCtx) ((globalCtx)->cameraPtrs[(globalCtx)->activeCamera])
 
-#define LINK_IS_ADULT (gSaveContext.linkAge == 0)
-#define LINK_IS_CHILD (gSaveContext.linkAge == 1)
+#define LINK_ADULT 0
+#define LINK_CHILD 1
```

Comment:
I would put this is z64save.h (where `linkAge` is defined), and make it a `LinkAge` enum. Also prefix should probably be `LINK_AGE_`

The comment on `SaveContext#linkAge` could then be removed entirely (enums for specific fields often have the same name as the field)

Reply:
Makes sense. Let me know if this is better.